src/components/MyAvatar.js

```diff
@@ -0,0 +1,16 @@
+import { Avatar } from '@mui/material';
+import { useAuthContext } from '~/contexts/AuthContext';
+import useImage from '~/hooks/useImage';
+
+export default function MyAvatar({ ...other }) {
+  const { session } = useAuthContext();
+  const imageUrl = useImage(session?.imageId);
+  const defaultImageUrl = '/images/logo.png';
+  return (
+    <Avatar
+      src={session ? imageUrl : defaultImageUrl}
+      alt={session ? `${session.firstName} ${session.lastName}` : 'logo'}
+      {...other}
```

Comment:
unclear naming

Reply:
Let me check again.

Reply:
I'm not sure about naming it.
Can you suggest?

Reply:
as you want pass remaining props then name it `props`

Reply:
I'm clear about that.